coach/src/concept-coach/index.cjsx

```diff
@@ -123,13 +123,16 @@ class ConceptCoachAPI extends EventEmitter2
       componentModel.channel.emit('close.clicked')
 
     @close = props.close
+
+    # Needs to be added on initialize since opening from a coach path calls
+    # initialize, and not open.  On will handle multiple logging in and out.
+    # 
+    # Wait until our logout request has been received and the close
+    User.channel.on('logout.received', @close)
+
     @component = coachWrapped.render(mountNode, props)
```

Comment:
[stopModelChannels](https://github.com/openstax/tutor-js/blob/master/coach/src/concept-coach/index.cjsx#L68-L70) offs the listeners.  Webview calls the `destroy` as the opposite of `initialize`.  I'll check a little more carefully.  I thought about the `off` when changing from `once` to `on`, but it needed the `on`, for like, if you logout, login, and then logout again.